resources/user-agents/browsers/android-webview/android-webview-generic.json

```diff
@@ -15,13 +15,6 @@
         "Comment": "Android WebView Generic"
       },
       "children": [
-        {
-          "match": "Mozilla/5.0 (#PLATFORM# Android/* Release/* Browser/applewebkit* Build/*) applewebkit* (*khtml*like*gecko*)*Version/*Chrome*Safari*",
```

Comment:
The only cases of this style UA I found were with Samsung devices, so I moved this pattern (but without a lot of the junk in the middle) to the Samsung Browser files as a WebKit version of the browser.

I did find some mention on wikipedia about Samsung using a WebKit version of the browser, and then moving to Chrome (https://en.wikipedia.org/wiki/Samsung_Internet_for_Android).

I looked through browscap's commit history to see when these patterns were first added, but there's no mention of why they were added, and no UAs hit them until a coverage add I did (#1331).